Tests/SentryTests/Integrations/Performance/UIViewController/SentryUIViewControllerPerformanceTrackerTests.swift

```diff
@@ -209,63 +203,57 @@ class SentryUIViewControllerPerformanceTrackerTests: XCTestCase {
             self.advanceTime(bySeconds: 1)
             callbackExpectation.fulfill()
         }
-        assertSpanDuration(span: lastSpan, expectedDuration: 1)
+        try assertSpanDuration(span: lastSpan, expectedDuration: 1)
```

Comment:
Same here, there are some `XCTUnwrap`s in `assertSpanDuration` forcing it to be `throws`